content/projects/esg/main.md

```diff
@@ -3,9 +3,7 @@ title: EuroScienceGateway
 description: Improving computational workflows in Galaxy by maturing Pulsar Network and publishing computational workflows as FAIR Digital Objects.
 ---
 
-# EuroScienceGateway Project
-
-![eosc | EuroScienceGateway](./eosc_euro_science_gateway.svg)
+![eosc | Euro Science Gateway](./eosc_euro_science_gateway.svg)
```

Comment:
Seem to be reverting #1733, change back to EuroScienceGateway

Reply:
I would also keep the `# h1` header as otherwise there is not textual header on page.